Basic-Car-Maintenance/Shared/Models/Contributor.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Contributor.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Yashraj jadhav on 01/10/23.
+//
+
+import Foundation
+
+struct Contributor: Codable, Hashable , Identifiable {
+    let login: String
+    let id: Int
+    let nodeID: String
+    let avatarURL: String
+    let gravatarID: String
+    let url: String
+    let htmlURL: String
+    let followersURL: String
+    let followingURL: String
+    let gistsURL: String
+    let starredURL: String
+    let subscriptionsURL: String
+    let organizationsURL: String
+    let reposURL: String
+    let eventsURL: String
+    let receivedEventsURL: String
```

Comment:
I don't think we actually need all of these properties, because they're not being used